calculator.html

```diff
@@ -37,10 +37,15 @@
             height="0" width="0" style="display:none;visibility:hidden"></iframe>
 </noscript>
 <!-- End Google Tag Manager (noscript) -->
-
     <div class="container">
         <div>
+            <p>
+              <a href="./instruction.html">
+                <b>Instruction</b> 📖
```

Comment:
```suggestion
                <b>How to mine</b> 📖
```

Мб под таблицей по центру?

Reply:
> Мб под таблицей по центру?

Я смержу но ты потом доделай плз